Error package grifts is not in GOROOT

Hi,
cloning the cli repo and install it with `go install -v ./cmd/buffalo` I cannot run any grift (`buffalo task ...`). 
Simply:
```
$ buffalo demo_project
$ cd demo_project
$ buffalo task
```
and you get this error
```
.grifter/main.go:9:2: package grifts is not in GOROOT (/usr/local/go/src/grifts)
Usage:
  buffalo task [flags]

Aliases:
  task, t, tasks

Flags:
  -h, --help   help for task

ERRO[0000] Error: exit status 1 
```

If you try this steps with cli version v0.18.2 all works.

--- a/go.mod
+++ b/go.mod
@@ -12,6 +12,7 @@ require (
 	github.com/gobuffalo/events v1.4.2
 	github.com/gobuffalo/flect v0.2.5
 	github.com/gobuffalo/genny/v2 v2.0.9
+	github.com/gobuffalo/here v0.6.6 // indirect
 	github.com/gobuffalo/logger v1.0.6
 	github.com/gobuffalo/meta v0.3.1
 	github.com/gobuffalo/plush/v4 v4.1.11

